chip8.hpp

```diff
@@ -2,26 +2,29 @@
 #define CHIP8_HPP
 
 #include <stdint.h>
-#include <stack>
-#include <string>
 #include <chrono>
 #include <random>
-
 #include <SDL2/SDL.h>
 #include <boost/log/trivial.hpp>
 
 class Chip8 {
 public:
     Chip8();
+
+    void set_key(int i, int keyPressedEvent);
     
     bool loadROM(const std::string& romFilename);
+
+    uint32_t get_grapich(int i) {
```

Comment:
misspelt graphic

Reply:
[ccd4b9e](https://github.com/ahchaprazian/Chip8-Emulator/pull/2/commits/ccd4b9e2cb1dfd8e3afecf9316bf6d336329187f) corrected the name of get_graphics and moved the implementation to chip8.cpp